src/lib/ui/DeviceManager/index.tsx

```diff
@@ -1,14 +1,14 @@
 import type { ReactElement } from 'react'
 import type { CommonDeviceProperties, Device } from 'seamapi'
 
-import { useDevices, type UseDevicesParams } from 'lib/hooks/use-devices.js'
+import { useDevices, type UseDevicesParams } from 'lib/seam/devices/list.js'
 
 export type DeviceManagerProps = UseDevicesParams
 
 /**
  * Fetch, list, and manage devices.
  */
-export const DeviceManager = (props: DeviceManagerProps): ReactElement => {
+export default function DeviceManager(props: DeviceManagerProps) {
```

Comment:
I'm not sure we need to export default exports. What is the use case? Everything will be imported by named export right? I think we should keep `export const DeviceManager` style in that case.

Reply:
I'm used to the primary component being a default export. This is inline with how MUI does things too.

That said, if we want to avoid default exports, that's fine too, it can just be:

```ts
export function DeviceManager
```

I would really prefer to keep components as named `function`(s); however, as in JS, only functions are hoisted. That is, using `const` would force us to put dependencies at the top.

I'm used to writing (and reading) code top-down with each dependency coming after the above. IMO it helps readability more than using the shorter `const` for definitions.

Reply:
> That said, if we want to avoid default exports

Yes, I would like to essentially prohibit all default exports from the project, expect possibly on a special case-by-case bases. Probably for things that looks like assets, like the SVG components, css, images, "json" files written in TS, etc. In those cases, the export would not be named.

When default / named exports are mixed it leads to inconsistency and can break treeshaking.

> I would really prefer to keep components as named function(s); however, as in JS, only functions are hoisted. That is, using const would force us to put dependencies at the top.

We can standardize on named functions, but I don't think the hoisting argument is correct. `const` and `let` are hoisted and deps can be defined below the component that uses them, see here: https://github.com/seamapi/react/blob/main/src/lib/components/DeviceManager/DeviceManager.tsx#L11

Can you provide an example of what `function` lets us do that `const` does not? Otherwise it's just a style preference between arrow or function (can have that debate separately if that's the case).

💯 For deps below the main component though.

Reply:
> Can you provide an example of what function lets us do that const does not?

Ya, but it's super contrived: https://codesandbox.io/s/react-playground-forked-gpl5fj?file=/index.js

https://user-images.githubusercontent.com/11449462/234869020-1dd9087e-f73a-4b3e-8b83-162f8a775085.mp4

In practice (with a bundler), you're right, it'd be _mostly_ a style choice since the bundler should move everything around, so it's all defined when it's used anyway. I say mostly because I have run into `access before init` errors that went away after changing `const` -> `function`. May have been due to the project's dependency graph though. 🤷‍♂️

Reply:
Removed default exports in https://github.com/seamapi/react/pull/47/commits